jabgui/src/main/java/org/jabref/gui/entryeditor/citationrelationtab/CitationRelationsTab.java

```diff
@@ -150,10 +150,10 @@ private SplitPane getPaneAndStartSearch(BibEntry entry) {
         citedByHBox.setPrefHeight(40);
 
         // Create Heading Lab
-        Label citingLabel = new Label(Localization.lang("Cites"));
-        styleLabel(citingLabel);
-        Label citedByLabel = new Label(Localization.lang("Cited By"));
-        styleLabel(citedByLabel);
+        Label citingLabel = new Label(Localization.lang("References (cites)"));
+        styleLabel(citingLabel, Localization.lang("Works cited by the work at hand"));
+        Label citedByLabel = new Label(Localization.lang("Citations (this paper is cited by)"));
```

Comment:
Remove "this paper is" --> consistent with "cites"

Otherwiese, 

    References (this paper cites)

But this is too long IMHO